incubator/elasticsearch/templates/client-deployment.yaml

```diff
@@ -21,7 +21,8 @@ spec:
 {{ toYaml .Values.client.podAnnotations | indent 8 }}
         {{- end }}
     spec:
-      serviceAccountName: {{ template "elasticsearch.fullname" . }}
+      securityContext:
+        fsGroup: 1000
```

Comment:
Won't this break old clusters as well? I'm not entirely sure how Kubernetes does this, if it does a chown on the Volumes or something. Have you tested the described upgrade, by any chance?

Reply:
I haven’t tested an upgrade myself.
I will see how that goes.

And now I am wondering how we could do that with CI. :)

Reply:
That would be a really cool feature...

Reply:
I should go look at the CI setup sometime.

Reply:
I am not sure of the original reason for setting the gid here.
I will check and see if we actually need it.